### ISSUE TITLE
Multplile sprity() calls will create the same sprite

### ISSUE THREAD

**Issue by ZxBud** [sprity/sprity#38]
In a project, I have to create multiple sprites for different image size (like 1 for 16x16, 1 for 32x32 and so on)
In the same gulpfile i have multiple tasks to generate those sprites.
When i call a task which require the subtasks with
```
gulp.task('sprites', ['sprites_16', 'sprites_32', 'sprites_64', 'sprites_500'], function() {});
```
Every png is the same, and contains every image.

**Comment by stephanedemotte** [comment]
try gulp-sequence 
https://www.npmjs.com/package/gulp-sequence

**Comment by ZxBud** [comment]
@stephanedemotte It works, but also doing parallels tasks the resut is very mixed.
I think someone should check for work isolation.

**Comment by rayliao** [comment]
@ZxBud I also have such a problem, So I can only run tasks one by one to solve.

**Comment by vlastv** [comment]
+1

**Comment by aslansky** [comment]
Should be resolved with #40.
